### DISCUSSION TITLE
[FR]: hope the WeChat synchronization function can realize/improve the following functions:

### DISCUSSION THREAD

**Discussion by hoojun98** [Quorafind/Obsidian-Thino#705] [Ideas 想法]
### Feature Requested | 请求的功能

希望微信同步功能能实现/完善如下功能：
（1）转发的公众号文章能够实现同步。目前直接转发公众号文章是无法实现同步的，只能通过复制公众号文章的网址链接实现同步；
（2）同步的网址可以自动识别网址标题，将网址标题和网址以markdown格式进行同步；
（3）某条信息中带【#标签】的微信同步可以作为上一条同步的标签处理，而不是作为新增的单条同步内容。
（4）可以设置微信同步的内容保存在指定的文件夹/文件下，方便归类。

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

[a comment by an automated account is not shown]

**Comment by juestchaos** [comment]
在计划中，转移到 discuss 区域，看更多用户反馈

[a comment by an automated account is not shown]
